src/vendoring/utils.py

```diff
@@ -30,11 +30,12 @@ def run(command: List[str], *, working_directory: Optional[Path]) -> None:
     )
     while True:
         retcode = p.poll()
-        line = p.stdout.readline().rstrip()
+        if p.stdout is not None:
+            line = p.stdout.readline().rstrip()
 
-        if line:
-            with UI.indent():
-                UI.log(line)
+            if line:
+                with UI.indent():
+                    UI.log(line)
```

Comment:
This is the one change, to make MyPy happy. The poll can return None in theory (thought it won't in this case). Not sure why this was a poll in the first place?